core/o11y/src/lib.rs

```diff
@@ -459,10 +460,13 @@ pub fn reload_log_config(config: Option<&log_config::LogConfig>) {
     };
     match result {
         Ok(_) => {
-            println!("Updated the logging layer according to `log_config.json`");
+            info!("Updated the logging layer according to `log_config.json`");
         }
         Err(err) => {
-            println!("Failed to update the logging layer according to the changed `log_config.json`. Errors: {:?}", err);
+            error!(
+                "Failed to update the logging layer according to the changed `log_config.json`. Errors: {:?}",
+                err
+            );
```

Comment:
@nikurt You introduced `println!` here instead of logging in #8810 (which unnecessarily spams workspaces-rs output), was it intended? Can you shed some light?

Reply:
Well, the whole point of this message is to complain about failures in (re-)setting up the logging infrastructure, so using that same logging infrastructure to inform about this failure seems like a chicken-egg problem.

Reply:
@nagisa Fair point! I have replaced `warn` with `eprintln` and keep the `info`.